### PR TITLE
prod/build

### PR DESCRIPTION
**Pull request by rosendi**

### CODE REVIEW

File: config/webpack.config.prod.js [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+const paths = require('./paths');
+
+if (process.env.NODE_ENV !== 'production') {
+  throw new Error('Production builds must have NODE_ENV=production.');
+}
+
+module.exports = {
+  bail: true,
+
+  devtool: 'source-map',
```
Comment (radiansz) [inline config/webpack.config.prod.js]: Do we need them in production?

**Comment by zouxuoz** [comment]
Fix tests

**Comment by zouxuoz** [comment]
lgtm

**Comment by radiansz** [comment]
lgtm

**Pull request merged**